src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -343,6 +346,9 @@ function BaseOptionsSelector(props) {
         [allOptions, sections],
     );
 
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    useEffect(() => setSections(sliceSections()), []);
```

Comment:
I'm not 100% sure if we shouldn't just use this instead:

```js
    const [sections, setSections] = useState(sliceSections());
```

It will uglify the ordering of the `useState`/`useCallback` in the file, but does essentially the same.
Would appreciate your thought @akinwale.

Reply:
I think I prefer `useState` here, since it makes the sections available for the first render, instead of right after the first render which is what happens with `useEffect`.